clearwater-infrastructure/usr/share/clearwater/bin/generic_create_diameterconf

```diff
@@ -12,7 +12,7 @@
 # This file creates a freeDiameter config file, and appropriate
 # TLS keys, for either Ralf, Homestead or Cedar.
 
-usage="Usage: generic_create_diameterconf [homestead|ralf|cedar] DIAMETER_IDENTITY DIAMETER_REALM LISTEN_PORT SECURE_LISTEN_PORT INCOMING_CONNECTIONS CHECK_DESTINATION_HOST"
+usage="Usage: generic_create_diameterconf [homestead|ralf|cedar] DIAMETER_IDENTITY DIAMETER_REALM LISTEN_PORT SECURE_LISTEN_PORT INCOMING_CONNECTIONS CHECK_DESTINATION_HOST TW_TIMER where TW_TIMER >= 6"
```

Comment:
It is better to put any commentary on parameters (such as allowed values) beneath the parameter list. 

e.g. "The TW_TIMER parameter is the time of the diameter TW timer in seconds. Must be an integer number greater than or equal to 6"